### ISSUE TITLE
Proof of Keys Liquidity Week: 03-10 January 2021

### ISSUE THREAD

**Issue by m52go** [bisq-network/growth#211]
## Objective

Achieve 0.5 BTC in BTC/CAD trading volume during 03-10 January 2021 through a week-long combination of market-making coupled with CAD-specific posts, videos, and social/community efforts to draw attention.

**This event is currently proposed as specific to the CAD market, but if you would like to spearhead efforts in another market for this week, please speak up. It would be great to target 2-3 more at the same time!**

## Learning Objective

See if we can get outsized benefits from a market making initiative when coupled with region-specific communication efforts. "Outsized" is hard to define since there isn't really a control group, so for now we stick to our goal of 0.5 BTC of daily volume during the week of the initiative.

Also: observe how much of the volume generated during the week remains in the days and weeks that follow. If there is a drop, further efforts should be made to determine what caused the drop.

## Rationale

CAD is a long-standing, promising, English-speaking Bisq market with a strong desire for Bisq liquidity. There are trades in it almost every day despite the thin offer book. [Something similar was attempted before](https://github.com/bisq-network/growth/issues/43). This effort is similar, but will attempt to push for bigger results via more CAD-specific publicity.

### Market Making

_Make the market ready and attractive for users to take offers._

- Recruit market-makers to make an attractive offer book for the duration of the week.
- Target 15 bids and 15 asks live throughout week ranging the gamut of CAD payment methods:
  - Interac e-Transfer
  - Revolut
  - TransferWise
  - Cash deposit

**Please add a :+1: below if you will participate!**

### Publishing

_Reach out to Canadian users with CAD-specific insights and how-tos._

The following are items that should be prepared in the days leading up to 13 December:
- what distinguishes Bisq from other options to buy bitcoin in the CAD region
- how to use Coldcard with Bisq (Coinkite is Canadian company)
- how to use Bisq with Interac (video, maybe quick post too)
- how to use Bisq with Revolut or TransferWise, perhaps with ex-Canada traders in mind to broaden usage (video, maybe quick post too)

These items should be published throughout the week so to create maximum word-of-mouth and so that Bisq stays stay top-of-mind on target users throughout the initiative.

**More ideas and collaborators to produce these items are welcome!**

### Community

_Amplify the campaign by making sure everyone hears about it._

Increase splash by collaborating with key community members.
- Get sticky on /r/bitcoinca for the week (mods and users have been very generous to Bisq so far)
- Encourage BTCSessions to make a video on Bisq (unsponsored, unfiltered; he agreed! he was planning to do one anyway, but getting in touch helped to set a date that coincides well with this initiative)
- Podcast mentions, other mass-media tactic (need to research; ideas welcome)

**More suggestions for key community members or outlets we should work with are welcome!**

**Comment by viperperidot** [comment]
Great initiative! One thing though, I am not certain that Revolut is available in Canada as of yet.

**Comment by m52go** [comment]
I only quickly searched 'revolut cad' and it seemed to be offered but I didn't investigate.

**Comment by pazza83** [comment]
Revolut offers CAD payments for existing users. I can send and receive $CAD.

However Revolut is not yet available to Canadians. It is coming soon: https://www.revolut.com/en-CA

**Comment by Olugbo** [comment]
Great initiative. I'm in CA, happy to help any way I can

**Comment by pazza83** [comment]
> Please add a 👍 below if you will participate!

This is a great idea.

I gave it a thumbs up for the idea but I cannot participate.

There are no payment options for the CAD market. It is not easy for non residents to access Interac e-Transfer. If anyone knows any options please share.

Just tried experimenting with TW. I can take CAD TW offers but not make them as you need a local CAD bank. 

Maybe we could have SWIFT up and online for the 03-10 January 2021.

**Comment by m52go** [comment]
> Interac e-Transfer

Yes I believe virtually all trades in CAD are through Interac...was hoping other methods might be available to diversify and possibly expand interaction with other markets, but it seems that won't be possible just yet.

> SWIFT

SWIFT would of course be nice to have as soon as possible, but it might be better to have payment method in the wild for a little while before promoting it hard...in case there are issues, hard promotion could backfire.

**Comment by LeadManPL** [comment]
Good idea, but how do you plan to avid issues with blocking users Revolut accounts? I saw many posts spread over the internet, that revolut was blocking user accounts after they made several transactions. This was AFAIR related to payment description containing transaction IDs

**Comment by m52go** [comment]
@LeadManPL yeah I've heard about this a few times too, but Revolut remains heavily used on Bisq. Not sure. Feel free to open an issue or start a chat on [Keybase](https://keybase.io/team/bisq) or create topic on the [forum](https://bisq.community/) if you have any additional information or concerns.

**Comment by LeadManPL** [comment]
@m52go : I just realised, that thi issue is about using TransferWise not Revolut.

**Comment by m52go** [comment]
As covered on the Q4 2020 call, this event flopped. Market makers didn't show up, liquidity was thin, and as a result, it didn't make sense to do further promotions.
